src/Form/Fields/Formatters/UploadFormatter.php

```diff
@@ -73,6 +74,11 @@ function fromFront(SharpFormField $field, string $attribute, $value)
 
             $storage->put($storedFilePath, $fileContent);
 
+            if($field->getShouldOptimize()){
```

Comment:
We have to handle here exception, or at least a check that we deal with an image...

Reply:
> By default the package will not throw any errors and just operate silently